app/models/user.rb

```diff
@@ -1,4 +1,7 @@
+require 'SecureRandom'
+
 class User < ActiveRecord::Base
+  before_create :set_auth_token
```

Comment:
Gotcha, no problem. I can take a crack at the generation UI, when I get some time. I'll just add to this PR when I get to it.